app/src/main/res/values/strings.xml

```diff
@@ -84,4 +84,24 @@
     <string name="raumlöschenConfirm">Sind sie sicher, dass sie diesen Raum löschen wollen?</string>
 
 
+    <string name="verbrauchsrechner_Strompreispkwh">Strompreis pro kWh in Cent</string>
+    <string name="verbrauchsrechner_LeistungkWh">Stromverbrauch</string>
+    <string name="verbrauchsrechner_Zeit_Leistung">Zeit unter Last</string>
+    <string name="verbrauchsrechner_Zeit_Standby">StandBy Zeit</string>
+    <string name="verbrauchsrechner_StandbyLeistung">StandBy-Verbrauch</string>
+    <string name="verbrauchsrechner_verbrauchjahr">Verbrauch/Jahr</string>
+    <string name="verbrauchsrechner_kostenJahr">"Kosten/ Jahr "</string>
+    <string name="verbrauchsrechner_beispiel_stromkosten">z.B. 7.6</string>
+    <string name="verbrauchsrechner_beispiel_stromverbrauch">z.B.50</string>
+    <string name="verbrauchsrechner_beispiel_vollastzeit">z.B. 4</string>
+    <string name="verbrauchsrechner_beispiel_StandByverbrauch">z.B. 0.5</string>
+    <string name="verbrauchsrechner_beispiel_StandByZeit">z.B. 20</string>
+
+
+    <string name="Einheit_kWh">kWh</string>
```

Comment:
According to Android naming convention this should be all snake_case.

Reply:
Will be fixed after the merch